core/inject.js

```diff
@@ -389,18 +370,23 @@ Blockly.inject.bindDocumentEvents_ = function() {
  * @private
  */
 Blockly.inject.loadSounds_ = function(pathToMedia, workspace) {
-  workspace.loadAudio_(
-      [pathToMedia + 'click.wav'], 'click');
-  workspace.loadAudio_(
-      [pathToMedia + 'delete.wav'], 'delete');
+  var audioMgr = workspace.getAudioManager();
+  audioMgr.load(
+      [pathToMedia + 'click.mp3',
```

Comment:
Yeah we converted our sounds to those different formats and added them to the repo.